docs/apm/service-overview.asciidoc

```diff
@@ -2,4 +2,106 @@
 [[service-overview]]
 === Service overview
 
-Selecting a <<services,*service*>> brings you to the *Service overview*.
\ No newline at end of file
+The *Service overview* page contains a wide variety of charts and tables that provide
+visibility into how a service performs across your infrastructure. To help you isolate
+potential issues, select a <<services,*service*>> to bring you to the *Service overview* page.
+
+[discrete]
+[[service-latency]]
+=== Latency
+
+Response times for the service. You can filter the *Latency* chart to display the average,
+95th, or 99th percentile latency times for the service.
+
+[role="screenshot"]
+image::apm/images/latency.png[Service latency]
+
+[discrete]
+[[service-traffic-transactions]]
+=== Traffic and transactions
+
+The *Traffic* chart visualizes the average number of transactions per minute for the selected service.
+
+The *Transactions* table displays a list of _transaction groups_ for the
+selected service and includes the latency, traffic, error rate, and the impact for each transaction.
+Transactions that share the same name are grouped, and only one entry is displayed for each group.
+
+By default, transaction groups are sorted by _Impact_ to show the most used and slowest endpoints in your
+service. If there is a particular endpoint you are worried about, click *View transactions* to view the
```

Comment:
Not sure about the wording here. I'd go with something more like "interested in" instead of "worried about", but the "View transactions" list takes you to the transactions overview which shows a similar list of transactions, and not a particular list of transactions.

Reply:
Thanks. I agree with you on the wording. I'll update it.